src/components/modals/DeleteCatalogModal.tsx

```diff
@@ -0,0 +1,66 @@
+import { Modal, Typography } from 'antd';
+import React, { useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useDeleteCatalog } from '../../hooks/catalog';
+import { useNotification } from '../../utils/NotificationContext';
+
+interface DeleteCatalogModalProps {
+  open: boolean;
+  closeModal: () => void;
+  catalog: string;
+}
+
+export function DeleteCatalogModal({
+  open,
+  closeModal,
+  catalog,
+}: DeleteCatalogModalProps) {
+  const navigate = useNavigate();
+  const { setNotification } = useNotification();
+  const mutation = useDeleteCatalog({
+    onSuccessCallback: () => {
+      setNotification(`${catalog} catalog successfully deleted`, 'success');
+      navigate(`/`);
+    },
+  });
+
+  const handleSubmit = useCallback(() => {
+    mutation.mutate(
+      { name: catalog },
+      {
+        onError: (error: Error) => {
+          setNotification(error.message, 'error');
+        },
+      }
```

Comment:
This is nice 👍  we should do the same error handling on all the delete assets.

Reply:
Thank you @jamieknight-db! 😄